js/gulpfile.js

```diff
@@ -46,13 +44,10 @@ for (const [target, format] of combinations([`all`], [`all`])) {
 // a minifier, so we special case that here.
 knownTargets.forEach((target) => {
     const umd = taskName(target, `umd`);
-    const cls = taskName(UMDSourceTargets[target], `cls`);
+    const esm = taskName(target, `esm`);
     gulp.task(`build:${umd}`, gulp.series(
-        `build:${cls}`,
-        `clean:${umd}`, `compile:${umd}`, `package:${umd}`,
-        function remove_closure_tmp_files() {
-            return del(targetDir(target, `cls`))
-        }
+        `build:${esm}`,
```

Comment:
@trxcllnt I don't know much about gulp but this seems to run the same job twice when I run `yarn build -t es5`. How can I avoid that?